packages/web/src/scss/components/Collapse/_Collapse.scss

```diff
@@ -6,17 +6,24 @@
     position: relative;
     height: 0;
     overflow: hidden;
+    visibility: hidden;
     transition-property: height;
     transition-duration: theme.$collapse-transition-duration;
     transition-timing-function: theme.$collapse-transition-timing;
 
+    &.is-open,
+    &.is-transition {
```

Comment:
why do we add this if it isn't used anywhere here? And shouldn't it be `is-transitioning`?

Reply:
This class is here for prevents 'blink' content when expanding and collapsing. Class is set only for expanding and collapsing. 
I will rename it